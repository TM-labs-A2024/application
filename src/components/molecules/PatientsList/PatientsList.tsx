import { List, Stack, Heading, Text } from '@chakra-ui/react'
import Patient from '@components/atoms/Cards/Patient'
import React from 'react'

export default function PatientsList({
  patients,
  pendingPatients,
  label
}: {
  patients: {
    href: string
    title: string
    description: string
    status?: string
  }[]
  pendingPatients: {
    href: string
    title: string
    description: string
    status?: string
  }[]
  label: string
}) {
  return (
    <Stack className="h-full">
      {(patients.length > 0 || pendingPatients.length > 0) && label && (
        <Heading as="h2" size="sm" className="mb-4">
          {label}
        </Heading>
      )}
      <div className="overflow-auto">
        {pendingPatients.length > 0 && (
          <>
            <Text size="sm" noOfLines={1} className="mb-4">
              Pendientes
            </Text>
            <List>
              {pendingPatients.map((patient) => (
                <div
                  className="border-b border-black py-4 last:border-transparent"
                  key={patient.href}
                >
                  <Patient
                    href={patient.href}
                    title={patient.title}
                    description={patient.description}
                    status={patient.status}
                  />
                </div>
              ))}
            </List>
          </>
        )}
        {patients.length > 0 && (
          <>
            {pendingPatients.length > 0 && (
              <Text size="sm" noOfLines={1} className="mb-4">
                Todos
              </Text>
            )}
            <List>
              {patients.map((patient) => (
                <div
                  className="border-b border-black py-4 last:border-transparent"
                  key={patient.href}
                >
                  <Patient
                    href={patient.href}
                    title={patient.title}
                    description={patient.description}
                    status={patient.status}
                  />
                </div>
              ))}
            </List>
          </>
        )}
      </div>
    </Stack>
  )
}
