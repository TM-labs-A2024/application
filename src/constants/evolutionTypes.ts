export const evolutionTypes = [
  {
    id: '1',
    name: 'Evolución'
  },
  {
    id: '2',
    name: 'Hospitalización'
  },
  {
    id: '3',
    name: 'Alta'
  },
  {
    id: '4',
    name: 'Primera consulta'
  }
]
