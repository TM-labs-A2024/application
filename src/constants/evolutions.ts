export const evolution = {
  specialty: {
    description: '',
    id: '',
    name: ''
  },
  author: '',
  title: '',
  description: '',
  'content-format': '',
  created_at: '',
  payload: {
    type: {
      value: '',
      label: ''
    },
    reason: '',
    description: '',
    history: '',
    examination: '',
    summary: '',
    diagnostic: '',
    comments: ''
  }
}
