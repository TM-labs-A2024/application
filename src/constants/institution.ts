export const institution = {
  id: '3fa85f64-5717-4562-b3fc-2c963f66afa1',
  govId: '123456',
  name: 'Clínica Tex Mex',
  credentials: '14578245',
  address: 'Av. Fernandez Peña, 147.',
  email: 'test@test.com',
  phoneNumber: '+582742214441',
  institutionUser: {
    id: '',
    firstname: '',
    lastname: '',
    govId: '',
    birthdate: '',
    email: 'test@test.com',
    phoneNumber: '+582742214441',
    role: '',
    institutionId: ''
  }
}
