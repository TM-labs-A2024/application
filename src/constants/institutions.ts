export const institutions = [
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa1',
    name: 'Clínica Tex Mex 1',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa11',
    name: 'Clínica Tex Mex 2',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa12',
    name: 'Clínica Tex Mex 3',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa13',
    name: 'Clínica Tex Mex 4',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa2',
    name: 'Clínica Tex Mex 5',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa3',
    name: 'Clínica Tex Mex 6',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa4',
    name: 'Clínica Tex Mex 7',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa5',
    name: 'Clínica Tex Mex 8',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
    name: 'Clínica Tex Mex 9',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa7',
    name: 'Clínica Tex Mex 10',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa8',
    name: 'Clínica Tex Mex 11',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441',
    pending: true
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66afa9',
    name: 'Clínica Tex Mex 12',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af10',
    name: 'Clínica Tex Mex 13',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af11',
    name: 'Clínica Tex Mex 14',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af12',
    name: 'Clínica Tex Mex 15',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af13',
    name: 'Clínica Tex Mex 16',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af14',
    name: 'Clínica Tex Mex 17',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af15',
    name: 'Clínica Tex Mex 18',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af16',
    name: 'Clínica Tex Mex 19',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  },
  {
    id: '3fa85f64-5717-4562-b3fc-2c963f66af17',
    name: 'Clínica Tex Mex 20',
    credentials: '14578245',
    address: 'Av. Fernandez Peña, 147.',
    email: 'test@test.com',
    phone: '+582742214441'
  }
]
