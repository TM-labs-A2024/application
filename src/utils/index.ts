import { formatDate } from './dates'
import { sendEmail, generateVerificationCode, setupEmailSending } from './email'
import { setupErrorNotification } from './notifications'
import { isWeb, isIOS, isAndroid, isMobile } from './platform'

export {
  isWeb,
  isIOS,
  isAndroid,
  isMobile,
  sendEmail,
  generateVerificationCode,
  setupEmailSending,
  setupErrorNotification,
  formatDate
}
