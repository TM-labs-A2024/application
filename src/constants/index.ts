import { doctors } from './doctors'
import { evolutionTypes } from './evolutionTypes'
import { institution } from './institution'
import { institutions } from './institutions'
import {
  ACCESS_REMOVED,
  FILTERS_APPLIED,
  ACCESS_REQUESTED,
  ATTACHMENT_DELETED,
  IMAGE_DELETED,
  ACCESS_DENIED,
  ACCESS_GRANTED,
  ACCESS_REMOVAL,
  GENERIC_NOTIFICATION,
  GENERIC_ERROR
} from './notifications'
import { nurse } from './nurses'
import { patient, sexOptions, statusOptions } from './patient'
import { patients } from './patients'
import { specialties, specialities } from './specialties'
import { specialtyData } from './specialtyData'

export {
  specialties,
  specialities,
  doctors,
  nurse,
  patient,
  sexOptions,
  statusOptions,
  specialtyData,
  evolutionTypes,
  patients,
  institution,
  institutions,
  ACCESS_REMOVED,
  FILTERS_APPLIED,
  ACCESS_REQUESTED,
  ATTACHMENT_DELETED,
  IMAGE_DELETED,
  ACCESS_DENIED,
  ACCESS_GRANTED,
  ACCESS_REMOVAL,
  GENERIC_NOTIFICATION,
  GENERIC_ERROR
}
