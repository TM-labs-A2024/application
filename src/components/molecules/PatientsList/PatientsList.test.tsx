import { patients } from '@src/constants'
import { render, screen } from '@test/utils/index'
import { formatDistanceToNowStrict } from 'date-fns'
import { es } from 'date-fns/locale/es'

import '@testing-library/jest-dom'
import PatientsList from './PatientsList'

const patientsFormatted = patients.map(
  ({ id, birthdate, govId, status, bed, firstname, lastname, pending }) => ({
    href: pending ? '/pacientes' : `/especialidades/${id}`,
    title: `${firstname} ${lastname}`,
    description: `C.I: ${govId}, ${formatDistanceToNowStrict(new Date(birthdate), {
      locale: es,
      roundingMethod: 'floor'
    })}${status ? `, Cama: ${bed}` : ''}`,
    status,
    pending
  })
)

const pendingPatients = patientsFormatted.filter((patient) => patient.pending)

const approvedPatients = patientsFormatted.filter((patient) => !patient.pending)

const props = {
  patients: approvedPatients,
  pendingPatients,
  label: 'label'
}

const noPendingProps = {
  patients: approvedPatients,
  pendingPatients: [],
  label: 'label'
}

const noApprovedProps = {
  patients: [],
  pendingPatients,
  label: 'label'
}

describe('Molecules > PatientsList test', () => {
  test('The component shows the label if any of the patients or pendingPatients lists have at least one entry', () => {
    render(<PatientsList {...props} />)

    expect(screen.getByText(props.label)).toBeInTheDocument()
  })

  test('The component shows the list of pending patients if this one has at least one pending patient', () => {
    render(<PatientsList {...props} />)

    expect(screen.getByText('Pendientes')).toBeInTheDocument()
  })

  test("The component doesn't show the list of pending patients if it's empty", () => {
    render(<PatientsList {...noPendingProps} />)

    expect(screen.queryByText('Pendientes')).not.toBeInTheDocument()
  })

  test('The component shows the list of approved patients if this one has at least one approved patient', () => {
    render(<PatientsList {...props} />)

    expect(screen.getByText('Todos')).toBeInTheDocument()
  })

  test("The component doesn't show the list of approved patients if it's empty", () => {
    render(<PatientsList {...noApprovedProps} />)

    expect(screen.queryByText('Todos')).not.toBeInTheDocument()
  })

  test('The quantity of children is the expected', () => {
    render(<PatientsList {...props} />)

    expect(screen.getAllByTestId('patient-card')).toHaveLength(13)
  })

  test('Matches the snapshot', () => {
    render(<PatientsList {...props} />)

    expect(screen.getByTestId('patients-list')).toMatchSnapshot()
  })
})
