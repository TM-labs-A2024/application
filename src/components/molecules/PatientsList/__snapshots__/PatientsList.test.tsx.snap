// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Molecules > PatientsList test Matches the snapshot 1`] = `
<div
  class="chakra-stack h-full css-8g8ihq"
  data-testid="patients-list"
>
  <h2
    class="chakra-heading mb-4 css-9f6g39"
  >
    label
  </h2>
  <div
    class="overflow-auto"
  >
    <p
      class="chakra-text mb-4 css-zvlevn"
    >
      Pendientes
    </p>
    <ul
      class="css-155za0w"
      role="list"
    >
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/pacientes"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885561, 26 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/pacientes"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885562, 25 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
    </ul>
    <p
      class="chakra-text mb-4 css-zvlevn"
    >
      Todos
    </p>
    <ul
      class="css-155za0w"
      role="list"
    >
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa3"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                Josefa Feliciana Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885563, 27 años, Cama: B-67
              </p>
            </div>
            <span
              class="css-1eveppl"
            >
              <svg
                class="chakra-icon chakra-icon css-vvjt0k"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.983,0a12.206,12.206,0,0,0-8.51,3.653A11.8,11.8,0,0,0,0,12.207,11.779,11.779,0,0,0,11.8,24h.214A12.111,12.111,0,0,0,24,11.791h0A11.766,11.766,0,0,0,11.983,0ZM10.5,16.542a1.476,1.476,0,0,1,1.449-1.53h.027a1.527,1.527,0,0,1,1.523,1.47,1.475,1.475,0,0,1-1.449,1.53h-.027A1.529,1.529,0,0,1,10.5,16.542ZM11,12.5v-6a1,1,0,0,1,2,0v6a1,1,0,1,1-2,0Z"
                  fill="currentColor"
                />
              </svg>
              <span
                class="css-1ny2kle"
              >
                hospitalizado
              </span>
            </span>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa4"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885564, 29 años, Cama: C-66
              </p>
            </div>
            <span
              class="css-1eveppl"
            >
              <svg
                class="chakra-icon chakra-icon css-vvjt0k"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.983,0a12.206,12.206,0,0,0-8.51,3.653A11.8,11.8,0,0,0,0,12.207,11.779,11.779,0,0,0,11.8,24h.214A12.111,12.111,0,0,0,24,11.791h0A11.766,11.766,0,0,0,11.983,0ZM10.5,16.542a1.476,1.476,0,0,1,1.449-1.53h.027a1.527,1.527,0,0,1,1.523,1.47,1.475,1.475,0,0,1-1.449,1.53h-.027A1.529,1.529,0,0,1,10.5,16.542ZM11,12.5v-6a1,1,0,0,1,2,0v6a1,1,0,1,1-2,0Z"
                  fill="currentColor"
                />
              </svg>
              <span
                class="css-1ny2kle"
              >
                hospitalizado
              </span>
            </span>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa5"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                Josefa Feliciana Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885565, 30 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa6"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885566, 31 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa7"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885567, 32 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa8"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                Josefa Feliciana Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885568, 33 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa9"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885569, 26 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa10"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                Josefa Feliciana Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885560, 26 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa11"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885511, 26 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa12"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885512, 26 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="patient-card"
          href="/especialidades/3fa85f64-5717-4562-b3fc-2c963f66afa13"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Feliciano Gutierrez García
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: E99885513, 26 años
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
    </ul>
  </div>
</div>
`
